.ebextensions/grab-current-index.config

```diff
@@ -3,22 +3,18 @@ commands:
     command: "mkdir -p /var/app/solr-index/index"
   00-mkdir-solr-index-log:
     command: "mkdir -p /var/app/solr-index/tlog"
-  04-chown-local-root:
-    command: "chown -R 999:999 /var/app/solr-index/"
 
 #env vars from config are not init'd when "commands" run, but are when
 # container_commands run
 container_commands:
   00-list-env:
     command: "env"
   01-grab-index:
-    command: "aws --region=us-west-2 s3 cp ${S3_INDEX_PATH} /var/app/solr-index/solr-index.tar.bz2"
+    command: "sudo aws --region=us-west-2 s3 cp ${S3_INDEX_PATH} /var/app/solr-index/solr-index.tar.bz2"
   02-extract-index-archive:
-    command: "cd /var/app/solr-index; tar -xvf /var/app/solr-index/solr-index.tar.bz2" 
+    command: "cd /var/app/solr-index; sudo tar -xvf /var/app/solr-index/solr-index.tar.bz2" 
```

Comment:
this will run these as root, which is probably fine.

`sudo su user -c "tar -C /var/app/solr-index -xvf ..."` might be slightly better

from `man tar`:
```
     -C directory, --cd directory, --directory directory
             In c and r mode, this changes the directory before adding the
             following files.  In x mode, change directories after opening the
             archive but before extracting entries from the archive.
```

Reply:
Okay

Reply:
I tried. Looks like this needs to run as root. 
[ec2-user@ip-10-60-2-32 app]$ sudo su webapp -c  "aws --region=us-west-2 s3 cp s3://solr.ucldc/indexes/production/2022/05/solr-index.2022-05-05-19_02_18.tar.bz2 /var/app/solr-index/solr-index.tar.bz2"
download failed: s3://solr.ucldc/indexes/production/2022/05/solr-index.2022-05-05-19_02_18.tar.bz2 to solr-index/solr-index.tar.bz2 [Errno 13] Permission denied: u'/var/app/solr-index/solr-index.tar.bz2.E9B8EDFA'